### ISSUE TITLE
Subtitle timing change

### ISSUE THREAD

**Issue by M3BAID** [jarnedemeulemeester/findroid#739]
Would be a great addition if you could add a subtitle timing changer so if the subtitle leading or lagging it can be adjusted to match the audio/video playback

**Comment by denzhaummer** [comment]
i totally agree, it's a must have feature. the content i just watched had a subtitle timing glitch and i had no choice to use glitchy subtitles

**Comment by pgetsos** [comment]
Came to the repo looking for this issue. It would be very useful indeed

**Comment by juxuanu** [comment]
What would it take to implement this? Does mpv has an option for that, and it'd just be plumbing it to UI?
